lib/flags.ml

```diff
@@ -108,20 +108,22 @@ let we_are_parsing = ref false
 (* Current means no particular compatibility consideration.
    For correct comparisons, this constructor should remain the last one. *)
 
-type compat_version = V8_2 | V8_3 | V8_4 | V8_5 | Current
+type compat_version = V8_2 | V8_3 | V8_4 | V8_5 | V8_6 | Current
 
 let compat_version = ref Current
 
 let version_strictly_greater v = match !compat_version, v with
-| V8_2, (V8_2 | V8_3 | V8_4 | V8_5 | Current) -> false
-| V8_3, (V8_3 | V8_4 | V8_5 | Current) -> false
-| V8_4, (V8_4 | V8_5 | Current) -> false
-| V8_5, (V8_5 | Current) -> false
+| V8_2, (V8_2 | V8_3 | V8_4 | V8_5 | V8_6 | Current) -> false
```

Comment:
Sure! I don't like the current style either. I was thinking that maybe we should rely on the default order of OCaml enumerated types (which is the order we want) but it is undocumented so I don't know if that would be considered bad practice. See: http://stackoverflow.com/questions/39667692/can-i-rely-safely-on-ocamls-built-in-order-for-enumerations